litmus/AArch64Compile_litmus.ml

```diff
@@ -517,6 +517,31 @@ module Make(V:Constant.S)(C:Config) =
     | _,RV (V128,_),_ ->
         assert false
 
+    let store_post memo v rA rB s = match v,s with
+      | V32, 0 ->
+        let rA,fA,fB = str_arg1 V32 rA in
+        { empty_ins with
+          memo=memo ^ " " ^ fA ^",["^fB^"]";
+          inputs=rA@[rB;]; reg_env=[rB,voidstar]@add_w rA; }
+      | V32, k ->
+        let rA,fA,fB = str_arg1 V32 rA in
+        { empty_ins with
+          memo=memo ^ sprintf " %s,[%s],#%i" fA fB k;
+          inputs=rA@[rB]; reg_env=[rB,voidstar;]@add_w rA; }
```

Comment:
As teh instruction writes into `rB`, this register has to be part of the instruction output (input and output are use for liveness computation, I do not thinks that omitting 'rB` in outputs is harmful, but who knows?).
```suggestion
         outputs=[rB;]; inputs=rA@[rB]; reg_env=[rB,voidstar;]@add_w rA; }
```

Reply:
Done